latest_cov_build.py

```diff
@@ -0,0 +1,26 @@
+import tarfile
+import sys
+import os
+import taskcluster
+try:
+    from urllib.parse import urlencode
+    from urllib.request import urlopen, urlretrieve
+except ImportError:
+    from urllib import urlencode, urlretrieve
+    from urllib2 import urlopen
+
+
+index = taskcluster.Index()
+queue = taskcluster.Queue()
+route = "gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt"
+taskId = index.findTask(route)['taskId']
```

Comment:
Since it's only used here, put the value here directly and remove the route variable.
Use single quotes everywhere for strings instead of double quotes.